assets/src/blocks/Articles/ArticlesBlock.js

```diff
@@ -1,114 +1,91 @@
 import {ArticlesEditor} from './ArticlesEditor';
 import {frontendRendered} from '../frontendRendered';
+import {renderToString} from 'react-dom/server';
```

Comment:
Can we move this dependency at the top of the file?

Reply:
Why does it matter?